src/index.js

```diff
@@ -40,6 +40,15 @@ const _init = Symbol('init');
 const _defineSetting = Symbol('_defineSetting');
 const _internalSet = Symbol('_internalSet');
 
+// In order to prevent an unrestricted number of event listeners on Process, we create the listeners here
+// to close the database on exit.
+const _instances = [];
```

Comment:
Why is this named `_instances` over `instances`?

Reply:
This could also be a static property on the `Enmap` class itself.

Reply:
I named it `_instances` to match with the definitions near it, and make it clear it's not part of the Enmap public-props itself when it's used.

Would it be better to make _instances a Symbol as above and use a static property?